### PR TITLE
txm: separate persistence from batching broadcast

### PR DESCRIPTION
**Pull request by aalu1418**
ticket: https://smartcontract-it.atlassian.net/browse/BCI-1456

### CODE REVIEW

File: common/txmgr/resender.go [hunk 1]
```diff
@@ -132,7 +131,7 @@ func (er *Resender[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) runLoop() {
 func (er *Resender[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) resendUnconfirmed() error {
 	enabledAddresses, err := er.ks.EnabledAddressesForChain(er.chainID)
 	if err != nil {
-		return errors.Wrapf(err, "EthResender failed getting enabled keys for chain %s", er.chainID.String())
+		return fmt.Errorf("EthResender failed getting enabled keys for chain %s: %w", er.chainID.String(), err)
```
Comment (jmank88) [inline common/txmgr/resender.go:134]:
```suggestion
		return fmt.Errorf("Resender failed getting enabled keys for chain %s: %w", er.chainID.String(), err)
```

File: common/txmgr/resender.go [hunk 2]
```diff
@@ -142,7 +141,7 @@ func (er *Resender[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) resendUnconfi
 		var attempts []txmgrtypes.TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]
 		attempts, err = er.txStore.FindTxAttemptsRequiringResend(olderThan, maxInFlightTransactions, er.chainID, k)
 		if err != nil {
-			return errors.Wrap(err, "failed to FindEthTxAttemptsRequiringResend")
+			return fmt.Errorf("failed to FindEthTxAttemptsRequiringResend: %w", err)
```
Comment (jmank88) [inline common/txmgr/resender.go:144]:
```suggestion
			return fmt.Errorf("failed to FindTxAttemptsRequiringResend: %w", err)
```

**Review by prashantkumar1982 — COMMENTED** [review]
Looks good!
Just the rename comments from Jordan need to be taken care of

**Review by samsondav — APPROVED** [review]

**Comment by aalu1418** [comment]
note: there are still other Eth/Evm prefixes that will be removed in a follow up ticket

**Review by samsondav — APPROVED** [review]

**Pull request merged**